roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -93,7 +93,9 @@ kiali_defaults:
       discovery_auto_threshold: 10
       discovery_enabled: "auto"
       enabled: true
-      is_core_component: false
+      component_status:
```

Comment:
And I just realized, you are adding a new "component_status" section - alphbetical ordering means move this above the `discovery_auto_threshold` - do there everywhere else, too, in the kiali_cr example yaml.